README.md

```diff
@@ -1 +1,21 @@
-# gsoc-2021-final-doc
\ No newline at end of file
+
+# GSOC 2021 CNCF Buildpacks Journey
+
+### Special Thanks
+I can't thank my mentor Natalie enough for giving the most explanatory answers to even the simplest questions during this program. Throughout this program, he has been with me in all kinds of problems inside and outside of GSOC. Apart from my mentor, I would like to thank all the Buildpacks supporters who helped me on this journey, especially Anthony, Javier, Yael, who are part of Buildpacks.
+
+### Introduce
+Hi everyone, I am Halil, 3rd year computer engineering student at Mugla Sitki Kocman University. This summer I experienced a very exciting and learnful program with GSoC.
+
+ 
+I will share some information about what I did and couldn't finish on time. This information may be helpful for other students who want to continue or use or extend my project. Within the scope of GSOC, I had the opportunity to contribute to many different aspects of the Buildpacks project.
+
+
+During the GSOC, I had the opportunity to contribute to the following PR's first hand:
+
+- **buildpacks/rfcs** - [Prepare Phase Draft RFC](https://github.com/haliliceylan/rfcs/pull/1) [UNCOMPLETED]
+- **buildpacks/rfcs** - [Project TOML Conveter RFC](https://github.com/buildpacks/rfcs/pull/182)
+- **buildpacks/lifecycle** - [Adds acceptance tests for builder](https://github.com/buildpacks/lifecycle/pull/648)
```

Comment:
Maybe it's worth describing (since this isn't merged yet) what work is left to do - i.e., updating the tests to run on Windows. You could mention that it's feature complete otherwise.